lib/mayaUsd/render/mayaToHydra/renderOverride.cpp

```diff
@@ -585,7 +587,7 @@ MStatus MtohRenderOverride::Render(const MHWRender::MDrawContext& drawContext)
         MStatus  status;
         MDagPath camPath = getFrameContext()->getCurrentCameraPath(&status);
         if (status == MStatus::kSuccess) {
-#ifdef MAYA_CURRENT_UFE_CAMERA_SUPPORT
+#ifdef MAYA_CURRENT_UFE_CAMERA_SUPPORT && UFE_V2_FEATURES_AVAILABLE
```

Comment:
Feels a bit awkward --- isn't it the case that MAYA_CURRENT_UFE_CAMERA_SUPPORT implies UFE_V2_FEATURES_AVAILABLE?